File: Simulacao_Go/sensor_SPS30.go
package main

import (
	"fmt"
	"math/rand"
	"time"
)

// simularLeituraSensor simula uma leitura do sensor SPS30, retornando dois valores, PM2.5 (µg/m³) e PM10 (µg/m³).
func simularLeituraSensor() (float64, float64) {
	var1 := rand.Float64() * 10
	var2 := rand.Float64()*10 + 10
	return var1, var2
}

func main() {
	// Simular leituras do sensor a cada segundo
	for i := 0; i < 10; i++ {
		var1, var2 := simularLeituraSensor()
		fmt.Printf("Leitura do sensor: %.2f, %.2f\n", var1, var2)
		time.Sleep(time.Second)
	}
}

File: Simulação Python/sensor_SPS30.py
import random
import time

def simular_leitura_sensor():
    # Simula uma leitura dO sensor SPS30, existem 2 valores, PM2.5 (µg/m³), PM10 (µg/m³)
    var_1 = random.uniform(0, 10)
    var_2 = random.uniform(10, 20)
    return var_1, var_2
    
# Simular leituras do sensor a cada segundo
# for _ in range(10):
#     valor = simular_leitura_sensor()
#     print(f'Leitura do sensor: {var_1}, {var_2}')
#     time.sleep(1)
